packages/suite/src/middlewares/suite/messageSystemMiddleware.ts

```diff
@@ -43,10 +44,19 @@ const messageSystemMiddleware =
                     enabledNetworks,
                 },
             });
-
             const categorizedValidMessages = categorizeMessages(validMessages);
 
+            const validExperiments = getValidExperiments(config, {
+                device,
+                transport,
+                settings: {
+                    tor: getIsTorEnabled(torStatus),
+                    enabledNetworks,
+                },
+            }).map(item => item.id);
+
             api.dispatch(messageSystemActions.updateValidMessages(categorizedValidMessages));
+            api.dispatch(messageSystemActions.updateValidExperiments(validExperiments));
```

Comment:
Sorry for noticing that so late, but now I'm trying to test it on mobile and just realized, that this is only in desktop middleware and it's missing in mobile middleware so it does not support mobile even though it looks like only this part is missing (I believe we don't need ExperimentWrapper for the initial implementation, we can add it together with some real experiments targeting mobile).